docs/manual/idl.rst

```diff
@@ -1,36 +1,34 @@
-Working with idl
+Working with IDL
 ================
 
-At the time of writing there is no official mapping from OMG IDL to Python. The solutions we came up with here are therefore not standardized and are thus not compatible with other DDS implementations. However, they are based purely on the standard library type-hinting functionality as introduced in Python 3.5, meaning that any Python tooling available that works with type hints is also compatible with our implementation. To generate initializers and nice string representations we use :py:mod:`dataclasses` standard library module. This is applied outside of the other IDL machinery, so if you want you can control immutability, equality checking or even use a different ``dataclasses`` representation, for example `runtype`_.
-
-All idl type support is contained within the subpackage `cyclonedds.idl`, allowing you to use it even in contexts where you do not need the full |var-project| ecosystem.
+All IDL type support is contained within the subpackage `cyclonedds.idl`, which enables you to use it even in contexts where you do not need the full |var-project| ecosystem.
 
+.. Note::
+   There is no official OMG specification for mapping IDL to Python. The solutions here are therefore not standardized and are not compatible with other DDS implementations. However, they are based purely on the standard library type-hinting functionality as introduced in Python 3.5, so that any Python tooling that works with type-hints is also compatible with our implementation. To generate initializers and string representations use the :py:mod:`dataclasses` standard library module. This is applied outside of the other IDL machinery, therefore you can control immutability, equality checking, or even use a different ``dataclasses`` representation, for example `runtype`_.
 
 Working with the IDL compiler
 -----------------------------
 
-You use the IDL compiler if you already have an idl file to define your types or if you require interoperability with non-Python projects. The ``idlpy`` library is built as part of the python package leveraging the scikit-build cmake integration. We will soon integrate a locator mechanism into ``idlc`` to retrieve the library location, since the ``idlpy`` library will be part of the python package installation and not on the `LD_LIBRARY_PATH` or equivalent. For now you can employ the same tactic as ``idlc`` will do eventually and use a hidden method:
+Use the IDL compiler if you already have an IDL file that defines your types, or if you require interoperability with non-Python projects. The ``idlpy`` library is built as part of the python package leveraging the ``scikit-build cmake`` integration. 
```

Comment:
Nice update, but my writings here were even more outdated then I thought! `scikit-build` has been removed quite a while ago, so it can just be "The idlypy library is built as part of the Python package".